docs/api/sm-api.md

```diff
@@ -152,8 +152,22 @@ returns
 TBD
 
 #### Filtering
-TBD
+Filtering allows you to select a subset of messages in any folder. The call to list all messages is appended with query parameters that specify which attributes of a message you want to filter and one or more conditions to filter on. The general format is:
```

Comment:
Should you maybe add an anchor link and add it to the top similar to Pagination?